src/components/layout/Wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './Wrapper.modules.css';
+
+function Wrapper(props) {
```

Comment:
I don't understand naming in this case. Wrapper for what?

Reply:
Wrapper keeps the main content in the center of the page

Reply:
OK, but maybe you can give concrete naming for this component?